### DISCUSSION TITLE
Changing the way BBT writes the file path for PDFs so it works on two machines?

### DISCUSSION THREAD

**Discussion by david-singerman** [retorquere/zotero-better-bibtex#3018] [Q&A] [answered]
I have Zotero on two different machines (my work iMac and my home Macbook) and currently BBT is set up to automatically export to two different files.

But I see on the Automatic Exporting page that it would be OK to use the same file, or at least the same file name, in Dropbox:

> If working in a group, this can be accomplished by repeating step 1 for each person into the same cloud-shared folder. When doing step 1, make sure to use the same filename as the original bib-file. This will overwrite the bib-file every time the library is refreshed.

The only issue is that because I use Zotero to store PDFs, the filenames will keep changing, from /Users/[imac-username]/Zotero/storage/xxxxxxx.pdf to /Users/[macbook-username]/Zotero/storage/xxxxxxx.pdf.

Is there a way to get BBT to replace /Users/[username]/ with "~"?

Thank you!

**Comment by retorquere** [comment, marked as answer]
[yes](https://retorque.re/zotero-better-bibtex/exporting/scripting/index.html#use--in-file-paths-to-avoid-the-bib-file-being-different-on-different-computers).

**Reply by david-singerman** [reply]
Wow! I never work with scripts so I didn't even think to look there. Thanks so much.
